scripts/parse_live.py

```diff
@@ -25,7 +25,8 @@
 
 import chdb
 import config
-import yamwapi as mwapi
+import yamwapi
+import mwapi
```

Comment:
We don't really want mwapi **and** yamwapi right? Why wasn't yamwapi enough?

Reply:
Oh I use mwapi here (and in CD) because I remember you've told me you want to change yamwapi to mwapi at some point. But yeah it seems to be redundant here and `query_pageids` can do the same thing.

Reply:
Oh right, yeah we do want to switch, but let's do it in one go in a separate PR. It shouldn't be a difficult change but it is not trivial either, so it's best not to mix it up with unrelated changes (plus, it's not needed for your project). Thanks for remembering though :)